lib/preboots/wrhs-models.js

```diff
@@ -14,16 +18,19 @@ module.exports = function models(app, options, done) {
   //
   // Get the regular config unless we are in prod where it doesnt exist
   //
-  var ensure = app.config.get('ensure') || options.ensure;
-  var datastar = new Datastar(app.config.get('database') || {
-    config: app.config.get('cassandra')
-  });
+  const ensure = app.config.get('ensure') || options.ensure;
+  const dynamoDriver = new DynamoDB(app.config.get('database'));
+  const s3 = new S3(app.config.get('s3')); // probably should be part of bffs.init promise/await
```

Comment:
should it? Can move as it feels a bit awkward here.

Reply:
@Swaagie yea this feels weird but I understand why (to make a single AWS liveness thing). But might feel cleaner just in bffs' preboot